flytepropeller/pkg/compiler/test/compiler_test.go

```diff
@@ -464,10 +460,6 @@ func runCompileTest(t *testing.T, dirName string) {
 				if !assert.NoError(t, err) {
 					t.FailNow()
 				}
-				// make sure real CR has randomized suffix
-				assert.Regexp(t, regexp.MustCompile("name-[bcdfghjklmnpqrstvwxz2456789]{20}"), flyteWf.Name)
-				// then reset for the sake of serialization comparison
-				flyteWf.Name = "name"
 
```

Comment:
The cr-name-scheme-change is now non-default, and [tests are added here](flytepropeller/pkg/compiler/transformers/k8s/workflow_test.go), so removing these checks.